routers/repo/http.go

```diff
@@ -167,13 +180,33 @@ func HTTP(ctx *context.Context) {
 					}
 					return
 				}
+
+				tokenUser, err := models.GetUserByID(token.UID)
```

Comment:
No need to get user by ID, you can just compare `authUser.ID != token.UID` on line 190